data/remnant/remnant 2 side missions.txt

```diff
@@ -144,3 +144,201 @@ mission "Remnant: Visit to Paradise 1"
+			`	"Well met, <first>. We were just going to the monthly Celebration of Remembrance. Would you like to accompany us?"`
+			choice
+				`	"Yes, I would."`
+					goto celebrationattendance
+				`	"What is that?"`
+					goto celebrationdefinition
+				`	"No thanks."`
+			label celebrationdeclined
+			`	She nods. "Well, may the Embers guide you, <first>." With that, she rejoins the stream of people as they vanish from sight.`
+				decline
+			label celebrationdefinition
+			`	"The Celebration of Remembrance is a gathering; we take a moment to remember where we have come from, appreciate where we are, and express hope for where we will be in the future." Her gestures take on a critical tone. "That sounds very formal, but it is more artistic than defined." She glances at the slowing flow of people. "It will be starting shortly, and I want to attend this month, so I should get moving. Would you like to join us?"`
+			choice
+				`	"Yes, I would."`
+				`	"No thanks."`
+					goto celebrationdeclined
+			label celebrationattendance
+			`	As the two of you join the flow of people, Taely gestures once again. "If you would like, I could provide you with some commentary and explanation, but it is more about feeling than doing anything specific. Together, we remember and celebrate through free-form dance mixed with singing and music. You are welcome to participate, but try not to disrupt or impede others in the way you join the dancing."`
+			choice
+				`	"No, that is alright. I will just experience it."`
+				`	"Yes, that would be appreciated."`
+					goto celebrationannotated
+			`	The crowd you have joined merges with other flows of people as you all stream onward, eventually coming out of the typical hallways and into a massive cave. On one side, a large opening, framed by rough stone and a handful of shield emitters, looks up towards the night sky. Stalagmites with matching stalactites above them ring the perimeter of the room, but there is no sound of drips or trace of lingering moisture here. While the walls are still rough, natural stone, the floor and all the stalagmites have been polished smooth. With a quick glance, you estimate that there are already over a hundred people in here, and more are still filtering in around you and through several other doors. Taely leads you through the crowd, frequently gesturing greetings to those she passes by.`
+			`	Without warning the room stills, and all the doors close. A flurry of gestures washes out from the center of the room, as if everyone is repeating what someone has said, but you cannot see anyone starting it. "Perimeter secure?" comes the gestured question, followed by an answering ripple from the edges of the room back toward the center: "All clear."`
```

Comment:
```suggestion
			`	Without warning, the room stills, and all the doors close. A flurry of gestures washes out from the center of the room as if everyone is repeating what someone has said, but you cannot see anyone starting it. "Perimeter secure?" comes the gestured question, followed by an answering ripple from the edges of the room back toward the center: "All clear."`
```

Reply:
Are you sure? This doesn't feel like a good flow with the comma there.

Reply:
I think both versions are correct in terms of grammar, so it is a question of which rhythm you prefer.

Reply:
"Without warning the room stills, and all the doors close."
"Warning the room stills," sounds like giving a warning to "the room stills", whatever a "room still" is.
The 'without' means no such warning was given.
There probably should be a comma there.

Reply:
> giving a warning to "the room stills"
I'm pretty sure [still can also be a verb](https://www.oxfordlearnersdictionaries.com/definition/english/still_4), in which case the comma isn't necessary.

Reply:
I think it could go either way still, but @warp-core convinced me here it works best with the comma. :)

Reply:
I don't follow warp-core's example. The lack of warning is in regards to the room stilling. 

To me, having a comma there breaks the flow. "Without warning." Well, without warning about what?

It's not two different ideas, it's not even two different things influencing each other. It's one thing.

Reply:
Without is a modifier to the phrase, not a part of it; the comma breaks up the phrase it's modifying.

"(((Without (warning the room stills)), (and all the doors close.))"

"(((Without (warning)), (the room stills), (and all the doors close.))"

Reply:
OK, well, at this point I kind of feel it's coming down to what is required by grammatical rules.

1. If a comma is grammatically *required* here, then I put it in.
2. If a comma is *not* required, then I would prefer to not have the comma as I feel it breaks the flow and the feel of what this detail is supposed to convey.

Reply:
I believe the comma is grammatically required.

Reply:
This does not feel conclusive, but the members of the writing team who have commented seem to accept without a comma, so I am leaving it out.